README.md

```diff
@@ -6,6 +6,62 @@
 
 An SDK that build protocol logics for Protocolink
 
+## How to contribute?
+
+### Initial Setup
+
+1. Fork this repository.
+2. Set up your development environment and install the required dependencies.
+
+### Contribution Steps
+
+1. **Create a New Logic:**
+  - Add a new folder under `src/logics` named after the protocol (protocolId), using Hyphenation format.
+  - Create a new logic file under `src/logics/{protocolId}`, named with the action name (logicId). Prefix it with `logic.` and use Hyphenation format. The file name should be `logic.{logicId}.ts`.
+
+2. **Implement the Logic:**
+  - Implement the Logic class in the logic file using PascalCase. Append `Logic` to the action name.
+  - Extend the `core.Logic` class.
+  - Implement the following interfaces based on the requirements:
+    - `core.LogicTokenListInterface`: Support for tokens list function.
+    - `core.LogicOracleInterface`: Quotation function.
+    - `core.LogicBuilderInterface`: Txdata encode function for interacting with contracts.
+
+  **Example:**
+  ```typescript
+  class LogicExampleLogic extends core.Logic implements
+    core.LogicTokenListInterface, core.LogicOracleInterface, core.LogicBuilderInterface {
+    // Your implementation here
+  }
+  ```
+
+3. **Reference Examples:**
+  - Explore reference examples for different categories:
+    - Lending: [Aave V3](src/logics/aave-v2/), [Compound V3](src/logics/comopound-v3/)
+    - Swap: [Uniswap V3](src/logics/uniswap-v3/), [ParaSwap V5](src/logics/paraswap-v5/)
+    - FlashLoan: [Aave V3](src/logics/aave-v3/), [Balancer V2](src/logics/balancer-v2/)
+  - More: [GO](src/logics/)
+
+4. **Unit Testing:**
+  - Write tests that won't change based on block number increments.
+  - Test files are in the same path as the logic files, named `logic.{logicId}.test.ts`.
+  - Write tests for functions like `getTokenList()` and `build()` as needed.
+
+5. **Integration Testing:**
+  - Write tests that interact with the Router contract in real time.
+  - Test files are in `test/logics/{protocolId}/{logicId}.test.ts`.
+  - Utilize Logic's quotation function to generate Logic Data for contract interaction.
```

Comment:
```suggestion
  - Utilize Logic's quotation functions to generate Logic Data for contract interactions.
```

Reply:
If there's only one function, should it be named in the plural form?

Reply:
Updated